source/manual/business-finder.html.md

```diff
@@ -0,0 +1,60 @@
+
+The business finder is driven by 'tags'. Any piece of content can be assigned any tag. There is no concept of a hierarchy or grouping; each tag is independent.
+
+In order to group tags into separate facets, such as "Sector / Business Area", we had to [define the groupings](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml) somewhere. They're currently in the content tagger.
+
+### Subscription list
+
+Users can [sign up to a custom mailing list](https://www.gov.uk/find-eu-exit-guidance-business/email-signup) tailored to their choices. They can select the facet values they want to subscribe to, and will be emailed according to their chosen frequency whenever content is newly tagged or existing tagged content is updated.
+
+Multiple facets are subscribed to with 'or' logic, as opposed to 'and' logic. For example, someone subscribing to a mailing list for "Accommodation" and "Aerospace" will be notified of any content changes associated with either tag, and not just content that is associated with _both_ tags.
+
+This was accomplished by adding a new field to TBC: `combine_mode: 'or'`. We [pass `combine_mode` to email-alert-api](https://github.com/alphagov/email-alert-api/pull/867) when creating the subscription, so that it knows to send alerts if _any_ of the chosen facets have content updates.
+
+Given that subscriptions could now be custom selected by the user, rather than curated by a content editor, we found that the subscription title became very long and difficult to read if selecting too many facets. For this reason, we [added `EmailAlertListTitleBuilder` for creating comma-separated titles](https://github.com/alphagov/finder-frontend/pull/1164) that are easier to read. In the spirit of optimising code for deletion, much of the code is in separate files to the main title logic.
+
+### Subscription frequency
+
+We [added a `default_frequency` parameter](https://github.com/alphagov/finder-frontend/pull/1104) to finder-frontend and email-alert-frontend so that we could change the default selected frequency from "Immediately" to "Daily". This was in response to user research, where we found the quantity of EU Exit related updates was quite overwhelming.
+
+Currently only the business finder is making use of this parameter, but we believe other finders could find a use for it in the future.
+
+### Pinning
+
+Currently disabled. More info TBC.
+
+## Applications affected
+
+The business finder modifications live across several repositories.
+
+### finder-frontend
+
+Added EmailAlertListTitleBuilder and default frequency logic.
+
+### email-alert-api
+
+Added [OrJoinedFacetSubscriberList](https://github.com/alphagov/email-alert-api/search?q=OrJoinedFacetSubscriberList&unscoped_q=OrJoinedFacetSubscriberList) which is currently only used by the business finder.
+
+### content-tagger
+
+TBC
+
+### search-api
+
+There is a lingering `and_facet` that is no longer being used, and should be removed.
```

Comment:
Could you add an explanation as to what the `and_facet` does and why it would be ok to remove?